pom.xml

```diff
@@ -489,6 +495,11 @@
                         </excludes>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <!-- Fixes missing plugin during CI builds -->
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-jar-plugin</artifactId>
+                </plugin>
```

Comment:
Let's try to keep separate concerns in separate PRs. Something like this is easy to review and approve in isolation. In practice it takes much longer to review and approve PRs when multiple concerns are conflated into the same work.

Reply:
Sure, I can move this. This fixed the fact CI was failing which is why I included it. I can put in a separate branch for the CI fixes.